website/src/components/molecules/quickstartDoc.js

```diff
@@ -0,0 +1,42 @@
+import styles from "../../css/molecules/quickstartDoc.module.scss";
+import React, {useEffect, useState} from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+const QuickstartDoc = ({children, embeds}) => {
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
```

Comment:
I guess this useEffect can be safely deleted. `isLoading` status is visible with the spinner line 24